### ISSUE TITLE
Write code to drive the ALB

### ISSUE THREAD

**Issue by ctm** [ctm/mb2-doc#554]
Use [rusoto](https://crates.io/crates/rusoto_elbv2) to create a CLI to allow me to update the ALB (#553) settings.

This is part of Continuous Deployment (#550) and is simply the requirement that I write a standalone (at least for now) CLI that will configure the ALB that I'll initially configure using the AWS console.

**Comment by ctm** [comment]
Today various table windows were closing (and then reopening) and I strongly suspect it's because of the ALB in front of `ws.devctm.com`. The good news is since the WebSocket url is not visible, I don't really need to run it through the ALB. I can just use explicit ports. It does, however, require me to think a little more about _exactly_ what I want this tool to do.

OTOH, I know that at minimum I'll want to be able to add a rule, so even if I don't know everything I need to do, I can still get my hands dirty with rusoto and make something that adds a rule.

**Comment by ctm** [comment]
FWIW, the idle timeout for the load balancer itself is set to 60 seconds and so far I haven't been able to find a way to specify an alternate timeout for when a connection is upgraded to a websocket. So, if I wanted to continue to run the websocket through the ALB, it should just be a matter of cranking up the timeout and/or adding a heartbeat.

The lobby already uses a 30 second heartbeat so that people who have connectivity issues never have much more than a minute of unavailability, and that explains why people weren't seeing their lobby connections closing and reopening.

Bottom line is that I will continue to plan on not using the ALB to route the websocket and simply have the routing info known to the lobby and tables, at least for the first release.

**Comment by ctm** [comment]
In light of my previous comment, I think the functionality I want this week for the "code to drive the ALB" is simply a CLI tool that inspects the rules, can insert a rule and delete a rule.

Assuming nothing higher priority pops up, I'll scaffold before I run this morning and then fill in the blanks after I get back.

**Comment by ctm** [comment]
I did scaffold `alb_tool`, but as I was reading up on how rusoto handles credentials, I realized I really can just use the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2-mac.html). So, I've installed that and once I've familiarized  myself with their `elbv2` subcommand, I'll close this issue and leave my (currently unneeded) scaffold in mb2's `alb-554` branch.

**Comment by ctm** [comment]
That was a mini nightmare. I had an ancient `AWS_REGION` environment variable that was making all my queries be for `us-east-1` instead of the default in my config file of `us-east-2`, so even though all the arns have the region baked into them, none of the output I was getting from the `aws elbv2` commands was telling me of this region mismatch. Once I used the `--debug` command line switch I could see the `us-east-1` argument that was being constructed and once I used `aws configure list` it pointed out that the value was coming from an environment variable.

Anyway, I can now execute, for exaample: `aws elbv2 describe-rules --listener-arn arn:aws:elasticloadbalancing:us-east-2:438436405137:listener/app/devctm/6f0ddf4892e05d00/5db8ae40146d4e53` and I get proper output. I also cleared out three ancient IAM users, created a `devctm-elbv2` role and got rid of my ancient root-level access key.

Whee!
